tests/publiccloud/ipa.pm

```diff
@@ -36,9 +52,9 @@ sub run {
         my $system_max_boot_time = 120;
         my $out                  = script_output('grep "^Startup finished in" ' . $ipa->{logfile});
         record_info('Startup time', $out);
-        die 'Fail to find boot time in log' unless $out =~ /Startup finished in (\d{1,4}\.\d{3})s \(kernel\) \+ \d{1,4}\.\d{3}s \(initrd\) \+ \d{1,4}\.\d{3}s \(userspace\) = (\d{1,4}\.\d{3})s/;
-        record_info('Kernel boot is too slow',         result => 'fail') if $1 > $kernel_max_boot_time;
-        record_info('Overall system boot is too slow', result => 'fail') if $2 > $system_max_boot_time;
+        my $startup_timings = extract_startup_timings($out);
+        record_info('Kernel boot is too slow',         result => 'fail') if $startup_timings->{'kernel'} > $kernel_max_boot_time;
+        record_info('Overall system boot is too slow', result => 'fail') if $startup_timings->{'system'} > $system_max_boot_time;
```

Comment:
```suggestion
        record_info('Overall system boot is too slow', result => 'fail') if $startup_timings->{'overall'} > $system_max_boot_time;
```
otherwise looks good to me.